python/dgl/mock_sparse/__init__.py

```diff
@@ -2,3 +2,4 @@
 from .diag_matrix import *
 from .sp_matrix import *
 from .elementwise_op_sp import *
+from .reduce import * # pylint: disable=W0622
```

Comment:
Rename it to `reduction.py` or `reduce_op.py`. Try to avoid turn off lint as much as possible.

Reply:
Done.